packages/@uppy/aws-s3-multipart/types/index.d.ts

```diff
@@ -43,6 +43,7 @@ export interface AwsS3MultipartOptions extends PluginOptions {
       opts: { uploadId: string; key: string; parts: AwsS3Part[]; signal: AbortSignal }
     ) => MaybePromise<{ location?: string }>
     limit?: number
+    shouldUseMultipart?: boolean | ((file: UppyFile, fileSize: number) => boolean)
```

Comment:
Why is `fileSize` the second argument? Doesn't it already live in `file`?

Reply:
It's provided as a convenience.

Reply:
I think it's cleaner to just have `file`. It's not some deep object, it's simply `file.size`. We can add another argument in the future when people start using it more and need more things or conveniences.

Reply:
I agree that if UppyFile always has a `.size`, then it's unnecessary/confusing/redundant to have a separate argument for it as well

Reply:
Changed.